frontend/src/modules/automation/components/automation-form.vue

```diff
@@ -139,8 +139,8 @@
 
 <script setup>
 import {
-  computed, defineProps, defineEmits, reactive, ref, watch,
-} from 'vue';
+  computed, defineProps, defineEmits, reactive, ref, watch, onMounted
+} from "vue";
```

Comment:
We should use `''` instead of '""' right? Here and below
Should we get a warning for this?